tests/integration/security/rbac/v2/basic/testdata/istio-rbac-grpc-rules.yaml.tmpl

```diff
@@ -0,0 +1,50 @@
+# istio-rbac-grpc-rules.yaml to enforce access control for gRPC services using Istio RBAC v2 rules.
+
+# For service a:
+# * Allow b to call a's Echo method.
+# * Do not allow c to talk to a (since NonExistingMethod does not exist).
+# * Allow d to call any methods of a.
+
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: access-a-grpc-echo
+  namespace: {{ .Namespace }}
+spec:
+  rules:
+    - paths: ["/proto.EchoTestService/Echo"]
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: not-access-a-grpc
```

Comment:
Do we need `not-access-a-grpc`? we can just remove this.

Reply:
Technically no, but I think we should have this ServiceRole here to demonstrate that `paths` work. `access-a-grpc-echo` and `not-access-a-grpc` have the same paths except at the method level.